public/expenses/edit.php

```diff
@@ -20,93 +20,106 @@
-                            <div class="form-group row">
-                                <label for="created_at" class="col-sm-2 col-form-label">Date</label>
-                                <div class="col-sm-10">
-                                    <input
-                                        id="created_at"
-                                        name="created_at"
-                                        type="date"
-                                        class="form-control"
-                                        value="<?php echo date('Y-m-d', strtotime($expense->created_at)) ?>">
-                                </div>
-                            </div>
-                            <hr>
-                            <!-- Submit button -->
-                            <div class="form-group row">
-                                <div class="col-sm-10">
-                                    <button type="submit" name="submit" value="submit" class="btn btn-primary">
-                                        Update
-                                    </button>
-                                </div>
-                            </div>
-                        </form>
+                </div>
+                <!-- Date -->
+                <div class='expense__form--content'>
+                    <label for="created_at" class='expense__form--content-header' >Date</label>
+                    <div>
+                        <input
+                            id="created_at"
+                            name="created_at"
+                            type="date"
+                            class='expense__form--content-input'
+                            value="<?php echo date('Y-m-d', strtotime($expense->created_at)) ?>">
                     </div>
                 </div>
-            </div>
+                <hr>
+                <!-- Submit button -->
+                <div class='expense__form--content'>
+                    <div>
+                        <button type="submit" name="submit" value="submit" class="btn btn-primary">
+                            Update
+                        </button>
+                    </div>
+                </div>
+            </form>
         </div>
-    </div>
+    </section>
+    <section class='footer' >
+        <?php include public_path("layouts/footer.php") ?>
+    </section>
 </div>
-<?php include public_path("layouts/footer.php") ?>
+<script>
+    let labelLetter = document.querySelector(".expense__form--content-header");
+    let inputs = document.querySelectorAll('.expense__form--content-input');
+    console.log(inputs);
+    inputs.forEach(input=>{
+        input.addEventListener("mouseover", function(e){
+          e.target.parentElement.parentElement.firstElementChild.style.letterSpacing = ".8rem";
+        })
+        input.addEventListener("mouseout", function(e){
+          e.target.parentElement.parentElement.firstElementChild.style.letterSpacing = ".3rem";
+        })
+    })
```

Comment:
This code fragment is duplicate with the fragment in the add  page and could be extracted to a Js script file and included in both pages.

Reply:
Done in next pull request (issue-16-Delete-Modal)